spacy/sklearn.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+import spacy
+
+
+__all__ = [
+    'SpacyTransformer',
+    'DOC', 'VECTOR', 'SENTENCES', 'ENTITIES', 'NOUN_CHUNKS',
+    'SENTIMENT', 'TOKENS'
+    ]
+
+
+DOC = 'self'
+VECTOR = 'vector'
+SENTENCES = 'sents'
+ENTITIES = 'ents'
+NOUN_CHUNKS = 'noun_chunks'
+SENTIMENT = 'sentiment'
+TOKENS = 'tokens'
+
+
+class SpacyTransformer(BaseEstimator, TransformerMixin):
+
+    def __init__(self,
+                 model=spacy.en.English(),
```

Comment:
Hm, okay. If you want exactly the semantics you've written, I would prefer to implement it with a class variable. The function would then take a `None` value, and switch to the global on None. Like this:

```python

    class SpacyTransformer(BaseEstimator, TransformerMixin):
        nlp = None
        def __init__(self, nlp=None, **etc):
            if nlp is None:
                if self.nlp is None:
                    self.nlp = spacy.lang.en.English()
```

I think this makes the reuse of the variable a bit clearer. The problem is that it doesn't really give the caller proper control of which NLP model is loaded. Can we just require a model in the init?

```python

    class SpacyTransformer(BaseEstimator, TransformerMixin):
        def __init__(self, nlp, **etc):
            self.nlp = nlp
```

I think the caller should think about how the NLP model is passed through.